test/test_foreach.py

```diff
@@ -341,34 +341,65 @@ def test_int_scalar(self, device, dtype):
             for foreach_bin_op, foreach_bin_op_, torch_bin_op in self.bin_ops:
                 tensors = self._get_test_data(device, dtype, N)
                 scalar = 3
-                expected = [torch_bin_op(t, scalar) for t in tensors]
 
-                res = foreach_bin_op(tensors, scalar)
+                if dtype == torch.bool and foreach_bin_op == foreach_bin_op == torch._foreach_sub:
```

Comment:
`foreach_bin_op == foreach_bin_op == torch._foreach_sub`? Occurs on the next line too